### ISSUE TITLE
Problem with newer user IDs in get_followers and get_friends

### ISSUE THREAD

**Issue by llsigerson** [mkearney/rtweet#115]
with the get_followers and get_friends functions, I'm occasionally getting user IDs that are 1-10 digits off of the actual ID. this only happens with newer user IDs that begin with 7-9 and are longer (18 digits),  and I can't for the life of me figure out why. Running the same query in Python instead returns the correct ID. If anyone else has found and solved this problem, it'd be great. I've built a workaround, but that raises its own issues.

**Comment by mkearney** [comment]
Can you provide code for a replicable example? The fix may have been as easy as fixing the `stringify_ids` parameter, but I don't have a way to test it yet.

**Comment by mkearney** [comment]
It looks like the above issue is making a difference on the returned object (previously returned numeric; now it returns character), which should fix the issue for you. But let me know either way!

**Comment by llsigerson** [comment]
Looks like it did! Wow! Thanks so much for the quick fix! I'll let you know if this issue shows up anywhere else. So, in simple terms, you just changed how R treats this specific type of JSON data from  Twitter?

**Comment by mkearney** [comment]
Excellent!

Actually, I changed a parameter used in the request to Twitter's REST API. Twitter encourages devs to request "stringified" IDs (b/c long integers create problems). The parameter was supposed to be `stringify_ids = TRUE` but I had entered `stringify = TRUE`. I'm like 95% sure this wasn't a problem before, so they must have made their API more strict.

The other solution is to use `bit64::as.integer`, but that's not yet a part of base R, so strings seem best for now.
